src/mirco_contactstatus.cpp

```diff
@@ -36,14 +36,21 @@ void MIRCO::ComputeContactNodes(std::vector<double> &xvf, std::vector<double> &y
 
 void MIRCO::ComputeContactForceAndArea(std::vector<double> &force0, std::vector<double> &area0,
     double &w_el, int nf, std::vector<double> pf, int k, double GridSize, double LateralLength,
-    double ElasticComplianceCorrection)
+    double ElasticComplianceCorrection, bool PressureGreenFunFlag)
 {
   force0.push_back(0);
   double sum = 0;
 #pragma omp parallel for schedule(static, 16) reduction(+ : sum)  // Always same workload -> Static!
   for (int i = 0; i < nf; i++)
   {
-    sum += pf[i];
+    if (PressureGreenFunFlag)
+    {
+      sum += pf[i] * pow(GridSize, 2);
```

Comment:
What is pf here? Is it the piecewise pressure acting on each "square"? Then I think this should be computed by `sum += pf[i] * pow(GridSize/LateralLength, 2);`

Reply:
If the point-based Green function is used, then `pf` is the force at each square. However, when the pressure-based Green function is used, `pf` is the pressure at each square. Therefore, to calculate the force, we multiply it with `gridsize^2` and sum it up.

Reply:
So grid size is the length of a "pressure cell", not the total domain size? Maybe a short comment or a different variable name would be good here, to avoid confusion.

Reply:
@isteinbrecher  yes, that is correct. The variable for the size total domain is `LateralLength`, and for each cell is `GridSize`. I will add relevant info in the doxygen documentation to avoid confusion.